.travis.yml

```diff
@@ -2,31 +2,29 @@ language: php
 
 sudo: false
 
-env:
-    COMPOSER_OPTIONS=""
-
 php:
     - 5.4
     - 5.5
     - 5.6
-    - hhvm
     - 7.0
+    - hhvm
 
 matrix:
     include:
         - php: 5.4
           env: COMPOSER_OPTIONS="--prefer-lowest"
-    allow_failures:
-        - php: 7.0
-
-env:
-    - SYMFONY_VERSION="~2.3"
+        - php: 5.4
+          env: $SYMFONY_VERSION="^2.8.0"
+        - php: 5.4
+          env: $SYMFONY_VERSION="^3.0.0"
```

Comment:
in other bundles, they always test a specific version, like `3.0.*`. if you define it like this, you will always test the latest version available, is this what you want?